### ISSUE TITLE
Detect if plugin is installed

### ISSUE THREAD

**Issue by andreafalzetti** [gitpod-io/gitpod#13092]

**Comment by andreafalzetti** [comment]
This issue is slightly related to https://github.com/gitpod-io/gitpod/issues/10924, if we could find a way to bring attention to a warning to the user, it could be re-used in other areas, such as the issue just mentioned.

An idea that I had was using the backend control center.

**Comment by akosyakov** [comment]
> We can do it by checking the folders inside /workspace/.config/(JetBrains|JetBrains-latest)/RemoteDev-<ID_CODE>/_workspace_<REPOSITORY_NAME>/plugins, and compare this list of folders name against the plugins listed in .gitpod.yml.

I don't think that checking plugins dir is good idea. Using API hooks should be safer.
